docs/tutorial.rst

```diff
@@ -953,6 +953,42 @@ The following example shows the form of these functions:
     ['aa', 'aaa', 'aab', 'aabb']
 
 
+Path Objects
+============
+
+Shells spend a lot of their time manipulating paths, so you may find it useful
+to handle paths as objects instead of string. A shorthand is provided to
+instantiate ``pathlib.Path`` objects:
```

Comment:
I also think this should come right before custom globbing section.  Also, it should `----` as the section underline, rather than `===`